test/scenarios-periodics/rhel-9.2-cncf-conformance.sh

```diff
@@ -0,0 +1,104 @@
+    local -r secondary_host_ssh_port=$(cat "${SCENARIO_INFO_DIR}/${SCENARIO}/vms/host2/ssh_port")
+    local -r primary_host_name="$(full_vm_name host1)"
+    local -r secondary_host_name="$(full_vm_name host2)"
+
+    scp -P "${primary_host_ssh_port}" "${ROOTDIR}/scripts/multinode/configure-pri.sh" "redhat@${primary_host_ip}":
+    ssh -p "${primary_host_ssh_port}" "redhat@${primary_host_ip}" ./configure-pri.sh "${primary_host_name}" "${primary_host_ip}" "${secondary_host_name}" "${secondary_host_ip}"
+
+    scp -3 -P "${primary_host_ssh_port}" \
+        "redhat@${primary_host_ip}:/home/redhat/kubelet-${secondary_host_name}.key" \
+        "redhat@${primary_host_ip}:/home/redhat/kubelet-${secondary_host_name}.crt" \
+        "redhat@${primary_host_ip}:/home/redhat/kubeconfig-${primary_host_name}" \
+        "redhat@${secondary_host_ip}":
+
+    scp -P "${secondary_host_ssh_port}" "${ROOTDIR}/scripts/multinode/configure-sec.sh" "redhat@${secondary_host_ip}":
+    ssh -p "${secondary_host_ssh_port}" "redhat@${secondary_host_ip}" ./configure-sec.sh "${primary_host_name}" "${primary_host_ip}" "${secondary_host_name}" "${secondary_host_ip}"
+
+    scp -P "${primary_host_ssh_port}" "redhat@${primary_host_ip}:/home/redhat/kubeconfig-${primary_host_name}" "${KUBECONFIG}"
+    export KUBECONFIG="${KUBECONFIG}"
+    echo "${primary_host_ip} ${primary_host_name}" | sudo tee -a /etc/hosts &>/dev/null
+}
+
+run_sonobuoy() {
+    # Configure cluster prerequisites
+    oc adm policy add-scc-to-group privileged system:authenticated system:serviceaccounts
+    oc adm policy add-scc-to-group anyuid     system:authenticated system:serviceaccounts
+
+    go install github.com/vmware-tanzu/sonobuoy@v0.56.16
+    ~/go/bin/sonobuoy gen \
+        --mode=certified-conformance \
+        --dns-namespace=openshift-dns \
+        --dns-pod-labels=dns.operator.openshift.io/daemonset-dns=default \
+        --e2e-parallel=y | oc apply -f -
+
+    # Wait for up to 5m until tests start
+    WAIT_FAILURE=true
+    for _ in $(seq 1 150) ; do
+        if ~/go/bin/sonobuoy status --json | jq '.status' &>/dev/null ; then
+            WAIT_FAILURE=false
+            break
+        fi
+        sleep 2
+    done
+    if ${WAIT_FAILURE}; then
+        echo "Failed to start tests after 5m"
+        exit 1
+    fi
+
+    # Use 1h timeout. A normal run on 2 CPUs takes 40-45min.
+    start=$(date +%s)
+    while [ "$(~/go/bin/sonobuoy status --json | jq -r '.status')" = "running" ] ; do
+        now=$(date +%s)
+        if [ $(( now - start )) -ge ${TIMEOUT} ]; then
+            echo "Tests running for 1h. Timing out"
+            break
+        fi
+        ~/go/bin/sonobuoy status --json | jq '.plugins[] | select(.plugin=="e2e") | .progress'
+        sleep 60
+    done
+
+    # shellcheck disable=SC2046  # Jq is unable to process escaped quotes
+    while [ $(~/go/bin/sonobuoy status --json | jq -r '."tar-info".name') == "" ] ; do
```

Comment:
We should probably add 5m timeout here as well. Otherwise, we may get stuck...

Reply:
Good catch!